### PR TITLE
Create Proposal.md

### PR DESCRIPTION
**Pull request by cpaluch**
Proposal for python library and Alexa skill workshop.

### Reference to issue
- #251 

### Description of the changes proposed in the pull request
- Proposal for the workshop. This proposal outlines the descriptions of pythons libraries and how building an Alexa Skill fits into this theme

### Link to Code SandBox
- link

### Link to Loom video (if blog/workshop completed)
- link

### Reviewers requested:
- @shreyagupta98 
- @atuljayaram

### CODE REVIEW

**Comment by atuljayaram** [comment]
Assigned to @AlexDoytch .

File: proposal.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+​	
+
+
+
+### 📌 Step 2
+
+👪 **TARGET AUDIENCE (3-5+ sentences):**
+The target audience of this workshop will be entry level programmers. They should have understandings of beginner level concepts like functions, lists etc but are looking to add more to their projects. I found that BitProject had many blogs and workshops in the works describing these libraries including an intoduction to pandas, flask and linear regression with scikit learn. This workshop's goal is to provide a description of pythons libraries to give beginners a launching point into one of the more technical blogs while also providing a fun project.
```
Comment (AlexDoytch) [inline proposal.md:23]: That is a lot of libraries to teach to entry level programmers. IS this going to be a couple.a series of blogs or is this going to be one long blog?

File: proposal.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+### 📌 Step 3
+
+> Outline your learning/teaching structure:
+
+**Beginning (2-3+ sentences):**
+The beginning of the workshop will open with descriptions of python libraries. This will include descriptions of data science, web development and machine learning using python to give beginners direction.
```
Comment (AlexDoytch) [inline proposal.md:30]: Will there be any interaction at this point?

File: proposal.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+**Middle (2-3+ sentences):**
+The middle section will be used to show how Python scripts can be written outside of the major categories to fulfill your ideas. This will lead into walking the user through creating their own alexa skill. The skill will be a roulette style game and will show the user the versatility of this language. 
+
+**End (2-3+ sentences):**
```
Comment (AlexDoytch) [inline proposal.md:35]: I liek the idea of letting the reader know what the intention is/the skills gained for the programmer.

File: proposal.md (hunk 1 above)
Comment (cpaluch) [reply proposal.md:23]: This is currently being developed as a workshop so over viewing these libraries and a technical walkthrough will fit within the time frame.

File: proposal.md (hunk 2 above)
Comment (cpaluch) [reply proposal.md:30]: The beginning will not be interactive, I did not see a good way to add brief interactions when describing multiple libraries without overwhelming beginners. Small examples that they are coding would likely make them more confused about these libraries. The goal of this section is to introduce a wide selection of libraries and describe what tools are available. Additionally, Bit project has a few technical blogs in the works on these topics that I will direct users to for more detailed explanations. (pandas, linear regressions with scikit learn and flask).

File: proposal.md (hunk 3 above)
Comment (cpaluch) [reply proposal.md:35]: Please let me know if you have any suggestions going further.

**Pull request merged**